go/vt/vttablet/endtoend/views_test.go

```diff
@@ -145,7 +148,30 @@ func TestDropViewDDL(t *testing.T) {
 	require.NoError(t, err)
 
 	// validate ZERO rows in _vt.views.
-	qr, err = client.Execute("select * from _vt.views", nil)
+	qr, err = client.Execute(qSelAllRows, nil)
 	require.NoError(t, err)
 	require.Zero(t, qr.Rows)
 }
+
+// TestGetSchemaRPC will validate GetSchema rpc..
+func TestGetSchemaRPC(t *testing.T) {
+	client := framework.NewClient()
+
+	viewSchemaDef, err := client.GetSchema(querypb.SchemaTableType_VIEWS)
+	require.NoError(t, err)
+	require.Zero(t, len(viewSchemaDef))
+
+	client.UpdateContext(callerid.NewContext(
+		context.Background(),
+		&vtrpcpb.CallerID{},
+		&querypb.VTGateCallerID{Username: "dev"}))
+
+	defer client.Execute(qDelAllRows, nil)
```

Comment:
nit: shoudn't we do this after `client.Execute` and before the `require.NoError`